pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -214,6 +274,44 @@ func (r *PerformanceProfileReconciler) getInfraPartitioningMode() (pinning apico
 	return infra.Status.CPUPartitioning, nil
 }
 
+func (r *PerformanceProfileReconciler) getContainerRuntimeName(profile *performancev2.PerformanceProfile) (mcov1.ContainerRuntimeDefaultRuntime, error) {
+	ctrcfgList := &mcov1.ContainerRuntimeConfigList{}
+	if err := r.List(context.Background(), ctrcfgList); err != nil {
```

Comment:
same, `context.Background()` is marginally better (if at all) than `context.TODO()` - not interruptable, and easy to miss when fixing all the contexts